README.md

```diff
@@ -21,18 +29,106 @@ $ kubectl create secret docker-registry iterativeai \
     --docker-password=<password>
 ```
 
-Create a file `override.yaml` with the following content or edit `values.yaml`
+### Prepare a TLS secret
 
-```yaml
-imagePullSecrets:
-  - name: iterativeai
+It is desired to enable Studio access via the `https` protocol
+(as opposed to `http`). This requires setting up
+a TLS secret for access (whether self-signed or "real").
+First, we'll need to generate a TLS/SSL certificate,
+and then, load it into a kubernetes secret in the namespace studio
+will be installed in.
+To create an SSL cert, you will need to know the domain (FQDN) that Studio
+will be accessible from (e.g. `studio.iterative.ai` for the official SaaS
+installation). 
+
+As an example, we'll assume your Studio installation will be available
+via: `https://my-studio.private.com`. We'll also use a self-signed
+certificate for the sake of simplicity. Of course, for a production
+installation we recommend using a certificate signed by a trusted CA.
+
+Let's create a self-signed SSL cert using `openssl` (you may need to install it):
+```bash
+openssl req \
+-x509 -newkey rsa:4096 -sha256 -nodes \
+-keyout tls.key -out tls.crt \
+-subj "/CN=my-studio.private.com" -days 365
 ```
 
-Deploy Studio
+This will create the files `tls.crt` & `tls.key` in your current dir.
+
+Now, let's create a TLS secret containing the contents from this cert:
 
+```bash
+kubectl create secret tls studio-ingress-tls \
+  --namespace studio \
+  --cert=tls.crt \
+  --key=tls.key
+```
+
+We will refer to this secret in the below installation instructions.
+
+### Install Studio
+
+Now, we are ready to deploy Studio using the Helm chart.
+
+Add the iterative helm repo:
 ```bash
 $ helm repo add iterative https://helm.iterative.ai
-$ helm install studio iterative/studio --namespace studio -f override.yaml
+```
+
+### Minimal Installation:
+
+To install studio with all default values (for sanity, testing), 
+Run the following command:
+```bash
+$ helm install studio iterative/studio \
+    --namespace studio \
+    --set-json='imagePullSecrets=[{"name": "iterativeai"}]'
+```
+
+### Functional Installation:
+
+Realistically, for a functional Studio app instance, you'll
+need to configure multiple values. In this example we'll prepare a
+more realistic and functional installation.
+
+Assumptions:
+- We have an ingress controller (nginx) installed on the cluster.
+- Studio will be available from the domain: `my-studio.private.com`
+  (we've registered this domain and made sure to redirect to our cluster).
+- We refer to the TLS secret `studio-ingress-tls` we've created in [a previous step](#prepare-tls-secret).
+
+Create a file named `studio-values.yaml`, with the following contents:
```

Comment:
```suggestion
Create a file named `values.yaml`, with the following contents:
```
Let's just use `values.yaml`

Reply:
When deploying Studio, do you think the user will have other `values.yaml` files? 

As we discussed earlier, the most common filename examples out in the wild are `values.yaml` and `override.yaml`

Reply:
if they follow the instructions they may very well have the values file from installing the nginx ingress controller.
This is why I specified the names. I don't see a clear downside on using the more specific name to be honest

Reply:
Makes sense. Let's keep it :+1: